### ISSUE TITLE
Question on Vtrace Training Result Stability

### ISSUE THREAD

**Issue by 1576012404** [google-research/seed_rl#17]
I train atari PongNoFrameskip-v4 using vtrace method，using the networks the same as the networks of dmlab(IMPALACNN and LSTM),with no FrameStack,get unstable training result as show in the figure.How to train to be stable?
with default parameters and change below:
`learing_rate:0.00048
adam_epsilon:3.125e-9
batch_size:32
num_actors:64
lambda_:0.95`
![1](https://user-images.githubusercontent.com/23149186/79727946-be690580-831f-11ea-9c8f-db9b2e6f1722.png)
![2](https://user-images.githubusercontent.com/23149186/79727955-c1fc8c80-831f-11ea-87c7-a131fbd5299f.png)
![2 2](https://user-images.githubusercontent.com/23149186/79727961-c4f77d00-831f-11ea-8aff-a08c95f71faa.png)
![3](https://user-images.githubusercontent.com/23149186/79727968-c759d700-831f-11ea-9d74-136233505852.png)

**Comment by lespeholt** [comment]
Lowering learning rate and lowering lambda may help. We haven't tuned V-trace much for Atari though.
